src/reflect/scala/reflect/internal/Definitions.scala

```diff
@@ -1140,6 +1148,7 @@ trait Definitions extends api.StandardDefinitions {
     // Meta-annotations
     lazy val BeanGetterTargetClass      = requiredClass[meta.beanGetter]
     lazy val BeanSetterTargetClass      = requiredClass[meta.beanSetter]
+    lazy val ConstructorTargetClass     = requiredClass[meta.constructor]
```

Comment:
should be `getClassIfDefined("scala.annotation.meta.constructor") so as not to crash with older standard libraries.